Added some suggestions from Ian Swett. (#22)
MikeBishop/dns-alt-svc: draft-nygren-httpbis-httpssvc.md

associated parameters (such as transport protocol and keying material
for encrypting TLS SNI).  It also provides a solution for the
inability of the DNS to allow a CNAME to be placed at the apex

> of a domain name.  It also provides a way to indicate that hostname

Reviewer: From a pure reading-flow point of view, can we do something other than "It also provides" twice in a row?
```suggestion
of a domain name.  Finally, it provides a way to indicate that the origin
```
